test/test_nestedtensor.py

```diff
@@ -8386,6 +8449,14 @@ def __torch_dispatch__(self, func, types, args=..., kwargs=None):
 
 COMPILE_FORWARD_SKIPS_AND_XFAILS = [
     *FORWARD_SKIPS_AND_XFAILS,
+    # select(): pending unbacked symints not in returned output (needs fix)
+    XFailRule(
+        error_type=torch._dynamo.exc.InternalTorchDynamoError,
+        error_msg="Pending unbacked symbols",
+        op_match_fn=lambda device, op: (op.full_name == "select"),
+        sample_match_fn=lambda device, sample: ("batch_dim" in sample.name),
+        name="broken_select_backward_unbacked",
+    ),
```

Comment:
This one is just code movement right.

Reply:
yep from compile backward -> compile forward xfails